### DISCUSSION TITLE
Latest News

### DISCUSSION THREAD

**Discussion by adt-apjc** [adt-apjc/YAP#156] [Announcements]
__YAP 1.1.0__ release introduces the concept of the demo catalogue as the landing page. 
The catalogue provides access to the Hello World demo and allows users to load a custom demo configuration. This initial release exposes more demos for users belonging to Cisco's internal network, and we are planning to release external demos soon based on interest in the tool.

If you have any ideas, please comment in [YAP Idea Space](https://github.com/adt-apjc/YAP/discussions/152). 
For other feedback, use [chat about anything](https://github.com/adt-apjc/YAP/discussions/141).

<hr /><em>This discussion was created from the release <a href='https://github.com/adt-apjc/YAP/releases/tag/'>v1.1.0</a>.</em>

**Comment by marumer** [comment]
YAP has been accepted in Devnet Code Exchange ([link](https://developer.cisco.com/codeexchange/github/repo/adt-apjc/YAP))

```
Thank you for submitting your repo for consideration for Code Exchange.
We reviewed your submission and think it will be valuable to our community.
It has been approved and is now publicly available within Code Exchange.

Published!
 
Thank you very much for sharing this on Cisco Code Exchange. We hope to see you more around DevNet!
Published URL for repo : https://developer.cisco.com/codeexchange/github/repo/adt-apjc/YAP
```

**Comment by marumer** [comment]
__YAP 1.1.1 has been released__  Fully compatible with previous demos, we advise to upgrade

This release addresses the failure in documentation generation when installing YAP on a remote server. The rest of the minor commits are cosmetic.

We added support for a locally stored demo catalogue in the frontend/my-public-assets volume directory.
The YAP catalogue discovery process searches order is:

A file named demoCatalog.json stored in frontend/my-public-assets volume, mapped to the frontend app/public/my-assets
The URL specified in REACT_APP_CATALOG, configured in frontend/.env
The URL https://wwwin-github.cisco.com/raw/APJ-GSP-ADT/YAP-Zoo/master/demoCatalog.json
If none can be access, YAP render the default tiles My Demos and Hello World
